client/blocks/importer/wordpress/upgrade-plan/use-default-hosting-details.tsx

```diff
@@ -0,0 +1,58 @@
+import { next, trendingUp, chartBar } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import cwvtechReportJson from './cwvtech-report.json';
+import type { ReactNode } from 'react';
+
+type DefaultHostingDetails = {
+	[ key: string ]: {
+		title: string;
+		description: string | ReactNode;
+		icon: JSX.Element;
+	};
+};
+
+const higherSpeedPercentage = Math.round(
+	( cwvtechReportJson[ 'WordPress.com' ].goodLCP - cwvtechReportJson[ 'WordPress' ].goodLCP ) * 100
+);
+
+const fastResponsePercentage = Math.round( cwvtechReportJson[ 'WordPress.com' ].goodFID * 100 );
+
+const wpcomMinutesDowntime = 0;
+const otherHostsAverageDowntime = 63;
+
+export const useDefaultHostingDetails = (): DefaultHostingDetails => {
+	const translate = useTranslate();
+
+	return {
```

Comment:
Refactor it from array to object, as suggested in https://github.com/Automattic/wp-calypso/pull/90717#discussion_r1603326181.